src/main/java/com/minecolonies/coremod/client/gui/WindowCitizen.java

```diff
@@ -682,4 +753,25 @@ private void fulfill(@NotNull final Button button)
 
         }
     }
+
+    private final class RequestWrapper
+    {
+        private final IRequest request;
+        private final int depth;
+
+        private RequestWrapper(final IRequest request, final int depth) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)